dbcon/joblist/expressionstep.cpp

```diff
@@ -398,8 +398,10 @@ void ExpressionStep::populateColumnInfo(SimpleColumn* sc, JobInfo& jobInfo)
 //XXX use this before connector sets colType in sc correctly.
 //    type of pseudo column is set by connector
         if (dynamic_cast<PseudoColumn*>(sc) == NULL)
```

Comment:
Do we really need this for a PseudoColumn that is a result of our internal UDAF which result types can't be a char.

Reply:
The line:
if (dynamic_cast<PseudoColumn*>(sc) == NULL)
asked the question if this is NOT a PseudoColumn. So yes, we need these lines for everything that is NOT a PseudoColumn.